lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -5869,6 +5934,15 @@ namespace Js
         return RecyclerNewEnumClass(this->GetRecycler(), EnumFunctionClass, JavascriptGeneratorFunction, type, scriptFunction);
     }
 
+    JavascriptAsyncFunction* JavascriptLibrary::CreateAsyncFunction(JavascriptMethod entryPoint, GeneratorVirtualScriptFunction* scriptFunction)
+    {
+        Assert(scriptContext->GetConfig()->IsES6GeneratorsEnabled());
```

Comment:
Yes, we need the generator functionality all available for the async functions. But right now generators are enabled by default still I will just keep it there for now.